### PR TITLE
[DBIO-1341] Chore: After Lab verified, can delete all services

### PR DESCRIPTION
**Pull request by luthfiazhari**
### JIRA LINK 🟦
- https://blocksphere2020.atlassian.net/browse/DBIO-1341

### Changelog/Description 🗒️
- Make props on Button for hover style tooltip
- update props data on DataTable components
- add flaging: cannot delete service if lab only have one service.
- make hover message if lab only have one service on delete button

### WARNING ⚠️
-

### CODE REVIEW

**Review by agustinustheo — APPROVED** [review]

File: src/views/Dashboard/Lab/Services/index.vue [hunk 1]
```diff
@@ -216,6 +230,14 @@ export default {
...
        this.idItemDeleted = item.id
        this.showDialog = true
+    },
+
+    async checkServiceCount() {
+       const res = await queryServicesCountByOwnerId(this.api, this.pair.address)
+       this.serviceCount = Number(res.toHuman())
+       if (this.serviceCount <= 1) {
+          this.activeButton = false
+       }
```
Comment (agustinustheo) [inline src/views/Dashboard/Lab/Services/index.vue:240]: Wait, can't you get this data from the Vuex store?

**Review by agustinustheo — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Lab/Services/index.vue (hunk 1 above)
Comment (agustinustheo) [reply src/views/Dashboard/Lab/Services/index.vue:240]: Try retrieving the lab services number from here: https://github.com/luthfiazhari/debio-frontend/blob/fdcfce85f5bdecdeeb928cb5a4f479b7f3d31af4/src/store/substrate/index.js#L427

**Review by bukandicki — CHANGES_REQUESTED** [review]

File: src/views/Dashboard/Lab/Services/index.vue (hunk 1 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Lab/Services/index.vue]: this issue has been worked on, got data from the store @agustinustheo

**Review by agustinustheo — APPROVED** [review]

**Comment by agustinustheo** [comment]
Closing because of inaction @luthfiazhari @bukandicki, feel free to reopen when ready @luthfiazhari.

**Review by bukandicki — APPROVED** [review]

**Pull request merged**